frontend/src/features/public-form/components/FormFields/FieldFactory.tsx

```diff
@@ -61,6 +63,10 @@ export const FieldFactory = memo(
         return <LongTextField schema={field} colorTheme={colorTheme} />
       case BasicField.YesNo:
         return <YesNoField schema={field} colorTheme={colorTheme} />
+      case BasicField.Dropdown:
```

Comment:
>  must have been removed again due to merge conflicts

Let's discuss this during next sync. We need to be very careful with merge conflicts 😰